memecity.engine.ui/UI/Overlay/OverlayBarItem.h

```diff
@@ -0,0 +1,17 @@
+﻿#pragma once
+#include "OverlayItem.h"
+
+class OverlayBarItem : public memecity::engine::ui::overlay::OverlayItem<int>
```

Comment:
ik zou bij deze class nog wat toevoegen. `max` en `width` vooral. Zo kan je wat dynamischer je bar opbouwen. Dan kan je berekenen hoelang het groene gedeelte moet zijn, en hoelang de bar moet zijn.

ook zou ik de twee kleuren als parameters meegeven, zodat die dynamisch geset kunnen worden. Zo kunnen we voor exp bijvoorbeeld geel en grijs gebruiken.

Reply:
de `size`, `x`, `y`, mogen in deze class, want de text item heeft deze niet nodig.

Reply:
ik denk dat een `width` ook handig zou kunnen zijn, dan kan de lengte los van de `size` zijn